src/Xamarin.Android.Build.Tasks/Utilities/AndroidResource.cs

```diff
@@ -11,26 +11,27 @@
 namespace Monodroid {
 	static class AndroidResource {
 		
-		public static bool UpdateXmlResource (string res, string filename, Dictionary<string, string> acwMap, IEnumerable<string> additionalDirectories = null, Action<TraceLevel, string> logMessage = null)
+		public static bool UpdateXmlResource (string res, string filename, Dictionary<string, string> acwMap, IEnumerable<string> additionalDirectories = null, Action<TraceLevel, string> logMessage = null, Action<string, string> registerCustomView = null)
```

Comment:
`CopyAndConvertResource.cs` also uses `UpdateXmlResource`, and could be changed to avoid the creation of a temp file.

```csharp
				if (AndroidResource.UpdateXmlResource (res, filename, acw_map)) {
					if (!modifiedFiles.Any (i => i.ItemSpec == destfilename))
						modifiedFiles.Add (new TaskItem (destfilename));
				}
```